utils/calibration.py

```
@@ -1,10 +1,12 @@
 """
 Calibration module for adjusting punch detection parameters
 """
+
 import cv2
 import numpy as np
```

Ruff (0.11.9)

`numpy` imported but unused

Remove unused import: `numpy`

(F401)